.vscode/autogpt-workspace.code-workspace

```diff
@@ -0,0 +1,69 @@
+{
+  "folders": [
+    {
+      "name": ".github",
+      "path": "../.github"
+    },
+    {
+      "name": ".vscode",
+      "path": "../.vscode"
+    },
+    {
+      "name": "assets",
+      "path": "../assets"
+    },
```

Comment:
These don't need separate interpreters or environments so can be covered by the root folder

Reply:
Fair drop them

Reply:
resolved